chsdi/views/wmtscapabilities.py

```diff
@@ -26,6 +26,15 @@ def getDefaultTileMatrixSet(tileMatrixSet):
     return tilematrixSet
```

Comment:
You could return the topLeftCorner directly from here. gargrid object has a `MAXX`, ` MAXY` attributes.

Reply:
Ok nice! Shouldn't it be `MAXY` and `MINX`?

Reply:
yes :D